projects/core/src/lib/operators/dialogs/rgb-composite/rgb-composite.component.html

```diff
@@ -0,0 +1,87 @@
+<geoengine-sidenav-header>Calculate Expression on Raster</geoengine-sidenav-header>
+
+<form [formGroup]="form" (ngSubmit)="add()" *ngIf="projectHasRasterLayers$ | async; else noDataInfo">
+    <geoengine-operator-dialog-container [loading]="loading$ | async | valueDefault : false">
+        <a mat-icon-button operatorInputIcon href="https://docs.geoengine.io/operators/expression.html" target="_blank">
+            <mat-icon>help_center</mat-icon>
+        </a>
+
+        <ng-container operatorInput>
+            <geoengine-multi-layer-selection
+                [types]="RASTER_TYPE"
+                formControlName="rasterLayers"
+                [min]="3"
+                [max]="3"
+            ></geoengine-multi-layer-selection>
+        </ng-container>
+
+        <a mat-icon-button operatorParametersIcon href="https://docs.geoengine.io/operators/expression.html" target="_blank">
```

Comment:
links to the expression

Reply:
[e41152d](https://github.com/geo-engine/geoengine-ui/pull/647/commits/e41152d0d1e578567e370d37b8d1d10456999d72)